media/pages/sk/nahlasit-chybajucu-instituciu/page.html

```diff
@@ -0,0 +1,3 @@
+{# vim: set filetype=htmldjango shiftwidth=2 :#}
+
```

Comment:
@martinmacko47 Co sa tyka style, moze byt alebo treba nieco upravit?

Reply:
@viliambalaz Silno sa mi nepaci, ze ten iframe ma vnorene vertikalne scrollovanie. Ale to sa zrejme neda vyriesit, kedze vnutorny dokument ma iny origin. Takze vonkajsi dokument nema ako javascriptom zistit rozmery vnutorneho dokumentu. Tak to teda asi nechajme tak.